install.sh

```diff
@@ -539,7 +541,7 @@ sudo_sh_c() {
 		echoh
 		echoerr "This script needs to run the following command as root."
 		echoerr "  $*"
-		echoerr "Please install sudo or su."
+		echoerr "Please install doas, sudo or su."
```

Comment:
`doas` is less popular than sudo, so let's list it last.

```suggestion
		echoerr "Please install sudo, su, or doas."
```